src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/exceptions.kt

```diff
@@ -0,0 +1,3 @@
+package com.bardsoftware.backend.fleet.rmarkdown
+
+class DockerProcessorException: Exception()
```

Comment:
Just put this exception next to `DockerProcessor` and remove this file.

Reply:
Done